yarn.lock

```diff
@@ -2584,6 +2584,10 @@
     call-me-maybe "^1.0.1"
     glob-to-regexp "^0.3.0"
 
+"@nicaea/types@link:types":
```

Comment:
This seems to have happened (again) after executing `yarn workspace joystream-cli add ...`

Reply:
I don't think it affects anything, so if yarn just keeps adding this entry perhaps we can leave it as is?